GitUI/Avatars/AvatarService.cs

```diff
@@ -55,7 +55,7 @@ public static void UpdateAvatarProvider()
                 BuildMainProvider(),
                 BuildFallbackProvider()
             }
-            .Where(p => p != null)
+            .WhereNotNull()
```

Comment:
Unfortunately, `Enumerable.Where<T?>` with a predicate that guarantees non-null values still returns `IEnumerable<T>`, not `IEnumerable<T?>`. We have a `WhereNotNull` extension method in the library that handles this case, avoiding use of `!` operator.